### DISCUSSION TITLE
Madara Community Call #17

### DISCUSSION THREAD

**Discussion by SamrendraS** [keep-starknet-strange/madara#1630] [General]
# Agenda
- Review of the rebase and current specifications (@tdelabro)
- Roadmap discussion for the orchestrator (@apoorvsadana)
- Roadmap for incorporating state root computations (@antiyro)
- Madara bootstrapper (@ocdbytes)
- SNOS update
- Update from Massa Labs on pending blocks: status and current progress

# Invitations
Google calendar: https://calendar.google.com/calendar/u/0/r/eventedit/OGFuN2s1NWhhZXVpcGgybmsxMTNhZDIzOG8gdGltb3RoZWVAc3Rhcmt3YXJlLmNv

Zoom: https://starkware-co.zoom.us/j/81001057592

# Date & Time
[Mon, 10 Jun 2024 at 4 PM CET](https://www.timeanddate.com/worldclock/converter.html?iso=20240610T140000&p1=195&p2=179&p3=1440&p4=176&p5=240)

**Comment by adrienlacombe** [comment]
gm, can you please record it? ty

**Reply by apoorvsadana** [reply]
Yes it will be recorded!

**Comment by byteZorvin** [comment]
The calendar link  is not working, can you check once?

**Reply by apoorvsadana** [reply]
Oh strange, will check it. But you can use the zoom link for joining the meet till then :)

**Reply by tdelabro** [reply]
Weird. It worked for me..

@apoorvsadana do you have the same issue?

**Reply by apoorvsadana** [reply]
Yep

**Comment by aniketpr01** [comment]
Please share the recording url or if it can uploaded to youtube channel
